service/src/test/java/bio/terra/workspace/common/fixtures/ControlledResourceFixtures.java

```diff
@@ -311,7 +312,9 @@ public static ControlledGcsBucketResource getBucketResource(String bucketName) {
         null,
         bucketName,
         /*resourceLineage=*/ null,
-        Map.of());
+        Map.of(),
+        "foo@gmail.com",
```

Comment:
Before this PR, these methods can be called by unit or connected tests. Now, they can only be used by unit tests.

For example, MockMvcUtils methods can be used by unit or connected tests. They are designed to work for both.

I don't think passing an argument is a big deal. I would rather do that than not have these methods work for connected.

Reply:
most of the connected test calls from the controller which calls sam to get the user email, but if it calls into the helper method and want to test createdByEmail, they can still modify it as makeDefaultControlledResourceFieldsBuilder returns a builder.

Reply:
If I'm understanding correctly, a connected test should not call these methods, because `"foo@gmail.com"` is not valid for connected tests. If you don't want to add a parameter, can you please add comments to these methods saying they should only be called by unit tests?